oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/trace/ProfileThreadSnapshotRecord.java

```diff
@@ -81,11 +80,7 @@ public ProfileThreadSnapshotRecord storage2Entity(final Convert2Entity converter
             snapshot.setDumpTime(((Number) converter.get(DUMP_TIME)).longValue());
             snapshot.setSequence(((Number) converter.get(SEQUENCE)).intValue());
             snapshot.setTimeBucket(((Number) converter.get(TIME_BUCKET)).intValue());
-            if (StringUtil.isEmpty((String) converter.get(STACK_BINARY))) {
-                snapshot.setStackBinary(new byte[] {});
-            } else {
-                snapshot.setStackBinary(Base64.getDecoder().decode((String) converter.get(STACK_BINARY)));
-            }
+            snapshot.setStackBinary(converter.getBytes(STACK_BINARY));
```

Comment:
For `ProfileThreadSnapshotRecord`,  I suppose the `entity2Storage` also needs some patch,

```diff
-converter.accept(STACK_BINARY, new String(Base64.getEncoder().encode(storageData.getStackBinary())));
+converter.accept(STACK_BINARY, storageData.getStackBinary());
```

Reply:
Fixed.